packages/react-instantsearch/examples/next-app/pages/index.js

```diff
@@ -26,7 +26,7 @@ export default class extends React.Component {
      to initialize the searchState. 
   */
   static async getInitialProps(params) {
-    const searchState = qs.parse(params.asPath.substring(2));
+    const searchState = qs.parse(asPath.substring(asPath.indexOf('?') + 1))
```

Comment:
Not sure to understand where does asPath came from. 

Also, what was the use case were the substring(2) was failing? 

Thanks for you PR @harshmaur!